### PR TITLE
Refactor redux actions in Policies widget

### PR DESCRIPTION
**Pull request by josemigallas**
**What this PR does / why we need it**:

Refactorization and cleanup of the redux actions and action creators.
* Rename actions with more (hopefully) clear names.
* Adds descriptions to each action
* Use actions/index as a proper index

**Verification steps**:
* `npm run update-dependencies`
* Navigate to `apiconfig/services/<id>/policies/edit`
* Policies widget should work as usual

**Note**:
This is part 10 of the refactorization of Policies:
[THREESCALE-2221: Policies: Normalise the use of policies schema and its types](https://issues.jboss.org/browse/THREESCALE-2221)

Previous step #1438.

### CODE REVIEW

File: app/javascript/src/Policies/actions/ThunkActions.jsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  addPolicyToChain,
+  removePolicy,
+  updatePolicyInChain,
+  fetchChain,
+  loadChain
+} from 'Policies/actions/PolicyChain'
+
+import { updatePolicyConfig } from 'Policies/actions'
+
+import type { UIComponent } from 'Policies/actions/UISettings'
```
Comment (damianpm) [inline app/javascript/src/Policies/actions/ThunkActions.jsx]: Why not importing in same line as `import { uiComponentTransition } from 'Policies/actions/UISettings'`

File: app/javascript/src/Policies/actions/ThunkActions.jsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  }
+}
+
+/**
+ * Hides the selected policy's detail, discarding any changes
+ * made in the form, then shows the policy chain component.
+ */
+export function closePolicyForm (): ThunkAction {
+  return function (dispatch: Dispatch) {
+    // TODO: dispatch(setSelectedPolicy(undefined))
```
Comment (damianpm) [inline app/javascript/src/Policies/actions/ThunkActions.jsx]: Why this TODO?

File: app/javascript/src/Policies/actions/ThunkActions.jsx (hunk 1 above)
Comment (josemigallas) [reply app/javascript/src/Policies/actions/ThunkActions.jsx]: This one is a type

Comment (damianpm) [reply app/javascript/src/Policies/actions/ThunkActions.jsx]: You are right

File: app/javascript/src/Policies/actions/ThunkActions.jsx (hunk 2 above)
Comment (josemigallas) [reply app/javascript/src/Policies/actions/ThunkActions.jsx]: I'll remove it

File: app/javascript/src/Policies/actions/ThunkActions.jsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  return function (dispatch: Dispatch) {
+    dispatch(loadRegistrySuccess(registry))
+    dispatch(loadChain(policiesConfig))
+  }
+}
+
+/**
+ * Fetch @registry_policies and @proxy.policies from rails OR call loadSavedPolicies
+ * if passed as arguments
+ * TODO: remove this method and call fetch OR loadSaved
```
Comment (didierofrivia) [inline app/javascript/src/Policies/actions/ThunkActions.jsx]: 👍

**Pull request merged**